### ISSUE TITLE
Issues with setup e-mail account

### ISSUE THREAD

**Issue by emillod** [nextcloud/mail#332]
### Steps to reproduce
1. Try to setup new e-mail

### Expected behaviour
Just want to setup new e-mail :)

### Actual behaviour
Nextcloud shows error with connecting to server

### Mail app

**Mail app version:** (see apps admin page)

**Mailserver or service:** My own mail on hosting

**Transport security - IMAP:** I'm trying all: None, SSL, TLS, STARTTLS 

**Transport security - SMTP:** I;m trying: None, SSL, TLS, STARTTLS

**Number of accounts:** 1

### Server configuration
**Operating system**: CentOS 7

**Web server:** Apache

**Database:** MySQL

And here i have logs from mail server:
Mar  6 12:11:04 mail3 postfix/postscreen[5502]: CONNECT from [**.**.**.**]:51414 to [**.**.**.**]:587
Mar  6 12:11:06 mail3 postfix/postscreen[5502]: HANGUP after 2.1 from [**.**.**.**]:51414 in tests before SMTP handshake
Mar  6 12:11:06 mail3 postfix/postscreen[5502]: DISCONNECT [**.**.**.**]:51414

Mar  6 13:37:45 mail3 postfix/postscreen[5502]: CONNECT from [**.**.**.**]:51424 to [**.**.**.**]:587
Mar  6 13:37:47 mail3 postfix/postscreen[5502]: HANGUP after 2.1 from [**.**.**.**]:51424 in tests before SMTP handshake
Mar  6 13:37:47 mail3 postfix/postscreen[5502]: DISCONNECT [**.**.**.**]:51424

**Comment by mckaygerhard** [comment]
i have same problem using courier and only SSL type connections..

**Comment by ChristophWurst** [comment]
Looks like the SMTP server requires the client to wait a bit. Have you tried applying a custom timeout? https://github.com/nextcloud/mail#smtp-timeout

**Comment by mckaygerhard** [comment]
right now i not could i'll test tomorrow, but seem also need prefixes are not well supported.. 

a mail system that not used prefixes all folders its a children toy! if this app does not support that, why for?

that's why free software are not marked place and every prject or makes private or goes to dead!

seems today working professional soltion its combined roundcube with owcloud-roundcube? 

and as final problem, only nextcloud 11? my provider does not permit dockers neither changes to webserver, use php 5.4 and this limit my usage to netxcloud 10, and also the api for nextcloud 10 are most knoleg either 11 that has some changes

in conclusion its very tedious the changes and "upgrdes" that are not necesary.. i starting using OC 6 and here more and more problems when i need a new featured, of course new feature vs not changes not upgrades.. 

so then please make it work with nextcloud 10 and prefixes! i swicht due oc was hav those problems and now i start xperience here too!

**Comment by ChristophWurst** [comment]
> so then please make it work with nextcloud 10 and prefixes! i swicht due oc was hav those problems and now i start xperience here too!

FYI we develop this app in our free time. If you think this is just a "children toy" then please use another software. You are very welcome to help us improve this app by implementing features, reviewing pull requests, testing releases and filing good bug reports. Thanks.

Also please take a look at [our code of conduct](https://nextcloud.com/code-of-conduct/).

**Comment by mckaygerhard** [comment]
oh yeah, imo why nextcloud born from outcloud...
